tests/test_xyz/test_points.py

```diff
@@ -0,0 +1,135 @@
+import itertools
+import pathlib
+from hypothesis import given, strategies as st
+
+import pandas as pd
+import numpy as np
+import pytest
+
+from xtgeo.xyz import Points
+import tests.test_common.test_xtg as tsetup
+
+PFILE = pathlib.Path("points/eme/1/emerald_10_random.poi")
+POINTSET2 = pathlib.Path("points/reek/1/pointset2.poi")
+POINTSET3 = pathlib.Path("points/battle/1/many.rmsattr")
+POINTSET4 = pathlib.Path("points/reek/1/poi_attr.rmsattr")
+CSV1 = pathlib.Path("3dgrids/etc/gridqc1_rms_cellcenter.csv")
+
+
+def test_custom_points():
+    """Make points from list of tuples."""
```

Comment:
No space after docstring in google convention (yes a bit ugly, but pydocstyle have no exceptions)

Reply:
Not sure if I follow, the google convention does indeed declare that there should not be any blank lines between docstring and code, but pretty much all of the other docstrings in the tests have a blank line before test begin. Am I missing something?

Reply:
I have just started pydocstyle compliance and implementing it gradually, so much code still not "there". Was thinking new code could adapt. That said, I personally like the blank lines so I am looking for exception; hence you can skip this proposal

Reply:
Ah, alright, then I understand. And tests aren't failing as we're doing this currently only for `xtgeo/plot` (settings pre-commit).. 

>Was thinking new code could adapt

I agree, but without any tests picking this up I think we'll end up checking in code that doesn't conform with `pydocstyle` (I'm running pre-commit as well, did not fail for me. Or maybe I enabled it after doing this specific commit? Never the less, should be picked up during CI).

At least for black I've experienced several times that for it to actually be included the repository would go through a "black all lines" commit eventually.. A bit more with pydocstyle I suppose, as it'll require more manual labour, but we could look into it :)